<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>968</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res.qrc">
    <normaloff>:/images/ressources/vgath-logo.png</normaloff>:/images/ressources/vgath-logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);
</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
       <item>
        <widget class="LeftInterface" name="leftInterface" native="true">
         <property name="styleSheet">
          <string notr="true">border: 1px solid #555;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FigureContainer" name="mainFrame" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="rightInterface" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="ToolsWidget" name="toolsWidget" native="true">
            <property name="styleSheet">
             <string notr="true">border: 1px solid #555;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(100,100, 100);</string>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveFileAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInterface"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Us"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionSave_File_As">
   <property name="text">
    <string>Save File As ...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionzz">
   <property name="text">
    <string>zz</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="text">
    <string>Save File As...</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionInterface">
   <property name="text">
    <string>Interface Overview</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FigureContainer</class>
   <extends>QWidget</extends>
   <header>Widgets/FigureContainer/figurecontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolsWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/ToolWidget/toolswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeftInterface</class>
   <extends>QWidget</extends>
   <header>Widgets/LeftInterface/leftinterface.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
